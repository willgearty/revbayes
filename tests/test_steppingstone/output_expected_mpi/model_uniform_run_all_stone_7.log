Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-26720.34	-26713.63	-6.710221	3.968599	0.0698729	0.0997969	0.699223	0.0318798	0.0729026	0.104221	0.105975	0.20821	0.159242	0.0252479	0.274633	0.110653	0.221451	0.0285431	0.192671	0.162288	0.259951	0.0535874	0.0973881	0.383208	0.107024	0.143056	0.390933	0.0357355	0.0891221	0.00492229	0.0499743	0.0531112	0.0340394	0.163415	0.184859	0.0315725	0.102421	0.00186617	0.201083	0.093447	0.0260999	0.0421824	0.123098	0.00907255	0.00607225	0.0747731	0.244452	0.159379	0.244067	0.132512	0.254355	0.183512	0.0261755	0.452629	0.774116	1.0844	1.68885	0.351467	0.312753	0.0733792	0.262401	0.4394317	5.573276
10	-26732.14	-26737.99	5.846147	1.510118	0.0755727	0.0346963	0.112884	0.0259833	0.144957	0.15768	0.0499343	0.0488309	0.042633	0.0354704	0.528589	0.0813027	0.063911	0.0861397	0.269196	0.125111	0.230369	0.0732447	0.17297	0.221142	0.0727677	0.216215	0.142508	0.055279	0.137588	0.00813278	0.00986511	0.051062	0.0327754	0.305112	0.0519865	0.00942915	0.0231056	0.00435439	0.304482	0.0802072	0.0209126	0.0603248	0.0457114	0.00666939	0.008869	0.0802162	0.162684	0.12392	0.156478	0.142853	0.0895516	0.462313	0.0248848	0.226881	0.590232	1.05101	2.13188	0.295453	0.433861	0.114688	0.155998	0.7367457	4.470871
