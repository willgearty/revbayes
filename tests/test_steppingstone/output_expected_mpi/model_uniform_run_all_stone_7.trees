Iteration	Posterior	Likelihood	Prior	psi
0	-26720.34	-26713.63	-6.710221	(Galeopterus_variegatus[&index=23]:0.390933,Nycticebus_coucang[&index=15]:0.192671,((Varecia_variegata_variegata[&index=22]:0.143056,(Lepilemur_hubbardorum[&index=11]:0.274633,Cheirogaleus_major[&index=4]:0.031880)[&index=41]:0.006072)[&index=33]:0.102421,(Macaca_mulatta[&index=13]:0.221451,((Chlorocebus_aethiops[&index=5]:0.072903,Otolemur_crassicaudatus[&index=16]:0.162288)[&index=25]:0.089122,((((Saimiri_sciureus[&index=20]:0.383208,((((Loris_tardigradus[&index=12]:0.110653,Tarsius_syrichta[&index=21]:0.107024)[&index=31]:0.184859,(Galago_senegalensis[&index=8]:0.208210,Lemur_catta[&index=10]:0.025248)[&index=28]:0.053111)[&index=42]:0.074773,(Colobus_guereza[&index=6]:0.104221,Hylobates_lar[&index=9]:0.159242)[&index=37]:0.026100)[&index=35]:0.201083,((Propithecus_coquereli[&index=19]:0.097388,Microcebus_murinus[&index=14]:0.028543)[&index=38]:0.042182,((Aotus_trivirgatus[&index=1]:0.069873,Perodicticus_potto[&index=18]:0.053587)[&index=43]:0.244452,Cebus_albifrons[&index=3]:0.699223)[&index=29]:0.034039)[&index=27]:0.049974)[&index=36]:0.093447)[&index=34]:0.001866,Daubentonia_madagascariensis[&index=7]:0.105975)[&index=40]:0.009073,Callicebus_donacophilus[&index=2]:0.099797)[&index=30]:0.163415,Pan_paniscus[&index=17]:0.259951)[&index=26]:0.004922)[&index=39]:0.123098)[&index=32]:0.031572)[&index=24]:0.035735)[&index=44];
10	-26732.14	-26737.99	5.846147	((Otolemur_crassicaudatus[&index=16]:0.125111,Chlorocebus_aethiops[&index=5]:0.144957)[&index=25]:0.137588,Galeopterus_variegatus[&index=23]:0.142508,((Perodicticus_potto[&index=18]:0.073245,(((Propithecus_coquereli[&index=19]:0.172970,Nycticebus_coucang[&index=15]:0.269196)[&index=34]:0.004354,(Tarsius_syrichta[&index=21]:0.072768,((((Cheirogaleus_major[&index=4]:0.025983,Varecia_variegata_variegata[&index=22]:0.216215)[&index=37]:0.020913,Hylobates_lar[&index=9]:0.042633)[&index=33]:0.023106,(Colobus_guereza[&index=6]:0.157680,Daubentonia_madagascariensis[&index=7]:0.049934)[&index=32]:0.009429)[&index=27]:0.009865,((Aotus_trivirgatus[&index=1]:0.075573,(Lemur_catta[&index=10]:0.035470,((Galago_senegalensis[&index=8]:0.048831,Lepilemur_hubbardorum[&index=11]:0.528589)[&index=31]:0.051986,Loris_tardigradus[&index=12]:0.081303)[&index=42]:0.080216)[&index=28]:0.051062)[&index=35]:0.304482,(((Pan_paniscus[&index=17]:0.230369,Macaca_mulatta[&index=13]:0.063911)[&index=40]:0.006669,Saimiri_sciureus[&index=20]:0.221142)[&index=41]:0.008869,Callicebus_donacophilus[&index=2]:0.034696)[&index=26]:0.008133)[&index=39]:0.045711)[&index=29]:0.032775)[&index=38]:0.060325)[&index=36]:0.080207,Microcebus_murinus[&index=14]:0.086140)[&index=24]:0.055279)[&index=43]:0.162684,Cebus_albifrons[&index=3]:0.112884)[&index=30]:0.305112)[&index=44];
