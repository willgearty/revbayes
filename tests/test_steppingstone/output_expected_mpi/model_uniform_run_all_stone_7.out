state	power	likelihood
4	0.00243	-27081.3
5	0.00243	-26817.8
6	0.00243	-26289.3
7	0.00243	-26363.5
8	0.00243	-26878
9	0.00243	-26365.2
10	0.00243	-26738
